### PR TITLE
feat: add veto tracking

### PR DESCRIPTION
**Pull request by Sekhmet**
This PR adds support for tracking veto status.

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (Sekhmet) [inline .prettierrc:1]: This codebase uses non-standard prettier style, adding this so it formats your code in way we want.

Comment (bonustrack) [reply .prettierrc:1]: Can't we use our eslint config?

**Review by bonustrack — APPROVED** [review]
utACK

File: .prettierrc (hunk 1 above)
Comment (Sekhmet) [reply .prettierrc:1]: We can, but it's totally different style so I'd need to refactor entire codebase here (which doesn't really fit into this PR).

**Pull request merged**